### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by Alex-Davis0**

### CODE REVIEW

**Comment by Alex-Davis0** [comment]
Code-journal 1
Code-journal 2

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+    entryId: data.nextEntryId,
+    title: $form.elements.Title.value,
+    photourl: $form.elements.photoUrl.value,
+    notes: $form.elements.Note.value
+
+  };
+
+  data.nextEntryId++;
+  data.entries.push(formvalue);
+  localStorage.setItem('data-profile', JSON.stringify(data));
```
Comment (BrettAlbright) [inline js/main.js]: make sure that all local storage functionality is completed in the data.ja file. Also, that functionality should match the functionality in the `javascript local storage` exercise. Local storage should only be used to load data at start, and store it at refresh.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
looking good, just some local storage updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**